travis/test_server.py

```diff
@@ -279,6 +279,13 @@ def hidden_line(line, main_modules, addons_path_list=None,
             # hidden all schema debug except
             #   dropped column, changed type and changed size
             return True
+    if 'superfluous' in line and 'dependency' in line:
+        superfluous_regex = re.compile(r'\.(?P<module>[\d\w$_]+): ')
+        superfluous_regex_search = superfluous_regex.search(line)
+        if superfluous_regex_search:
+            module = superfluous_regex_search.group('module')
+            if module not in main_modules:
+                return True
```

Comment:
I like for ALL except the official ones.

Look how I did it here:

https://github.com/Vauxoo/server-tools/blob/8.0/module_dependencies_view/models.py#L129

what do you think?